scripts/clickhouse_import_support/derived_table_construction_commands.sql

```diff
@@ -0,0 +1,36 @@
+DROP TABLE IF EXISTS sample_to_gene_panel_derived;
+DROP TABLE IF EXISTS gene_panel_to_gene_derived;
```

Comment:
Aren't these commands stored somewhere else?

Reply:
Yes. This file was a placeholder. I need to add calls to the script which processes the .sql files from the github repo (clickhouse.sql and materialized_views.sql) and bursts them into individual sql statements (1 per file). Also, we need to detect and skip over the special cases (genetic_alteration_derived, generic_assay_data_derived).

So this file will be going away